DataBase001/Model/Database.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DataBase001.Model
+{
+    
+    public class Database
+    {
+        public List<City> citys { set; get; }
+        public List<Route> routes { set; get; }
+        public List<Train> trains { set; get; }
```

Comment:
Citys -> Citites
Помимо этого если вне класса никто менять эти списки не собирается (именно присваиванием), то лучше это сразу запретить. через `readonly` или вот так

```suggestion
        public List<City> cities { private set; get; }
        public List<Route> routes { private set; get; }
        public List<Train> trains { private set; get; }
```